HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs

```diff
@@ -24,19 +25,43 @@ namespace GraphicalEditor
     /// <summary>
     /// Interaction logic for MainWindow.xaml
     /// </summary>
-    public partial class MainWindow : Window
+    public partial class MainWindow : Window, INotifyPropertyChanged
     {
         private Canvas _canvas;
         public List<MapObject> AllMapObjects { get; set; }
 
         private IRepository repository;
 
+        private Boolean _editMode = false;
+        public Boolean EditMode
+        {
+            get { return _editMode; }
+            set
+            {
+                _editMode = value;
+                OnPropertyChanged();
```

Comment:
Doesn't OnPropertyChange usually need to fire with the property name as argument? Not sure what happens without it.

Reply:
It looked weird for me too, but it works. I will probably add the property name to avoid any possible future problems.

Reply:
Sure, you have conflicts to resolve anyway, so no reason not to add it now.